OneSignalSDK/unittest/src/test/java/com/test/onesignal/MainOneSignalClassRunner.java

```diff
@@ -1377,7 +1342,6 @@ public void shouldUpdateNotificationTypesCorrectlyEvenWhenSetSubscriptionIsCalle
       assertEquals(1, ShadowOneSignalRestClient.lastPost.getInt("notification_types"));
    }
 
-
    @Test
    public void shouldAllowMultipleSetSubscription() throws Exception {
       OneSignalInit();
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.